packages/e2e/cypress/integration/common/shared.ts

```diff
@@ -288,6 +288,12 @@ Then('I see the button containing {string}', (name: string) => {
   }).should('exist');
 });
 
+Then('I do not see the button containing {string}', (name: string) => {
+  cy.findByRole('button', {
+    name: new RegExp(`${escapeRegExp(name)}`, 'i'),
+  }).should('not.exist');
+});
+
```

Comment:
It looks like this functionality already exists as ["I do not see the {string} button"](https://github.com/aws-amplify/amplify-ui/blob/feat-storage-browser/main/packages/e2e/cypress/integration/common/shared.ts#L305)